packages/build/src/plugins_core/deploy/buildbot_client.js

```diff
@@ -84,7 +84,18 @@ const getDeployDir = function ({ buildDir, repositoryRoot, constants: { PUBLISH_
 
 // We distinguish between user errors and system errors during deploys
 const handleDeployError = function (error, errorType) {
-  const errorA = new Error(`Deploy did not succeed: ${error}`)
+  const errorIs422 = error.includes('422')
```

Comment:
I wasn't sure whether the error had a status code but from logs I know it has 422. Please let me know if the error has a status code property.

Reply:
You're correct that we don't have a status code property here and only an error string and `error_type` which is used to distinguish between system and user errors:
- https://github.com/netlify/buildbot/blob/f4df34fdee5e467fb0a8a2f62e3840ffaba0a9bc/deployserver/deploy_server.go#L122-L126

I'm thinking however we should probably have it and I guess adding it wouldn't be a huge lift, what do you think? 👀 I'll leave it up to you though 👍

Reply:
Yeah, I agree.

Reply:
Is the type of error a json object?